setup.py

```diff
@@ -51,8 +51,7 @@
 DEBUG = os.getenv("DEBUG", "0") == "1"
 COVERAGE = os.getenv("COVERAGE", "0") == "1"
 
-# Customize the wheel plat-name, usually needed for MacOS builds.
```

Comment:
Is this no longer needed?

Reply:
This was most likely required before because python builds for macOS x86_64 on GHA were targeting versions of macOS that are too recent. All GHA pythons used on macOS arm64 are using the official universal2 installers that allows to get rid of this (with a slight exception for the 3.8 version as noted elsewhere).
I removed the comment but left the code in case some downstream projects are using this.